src/SFA.DAS.ApplyService.Application/Apply/Review/Return/ReturnRequestHandler.cs

```diff
@@ -76,14 +76,14 @@ private async Task NotifyContact(Guid applicationId, int sequenceId)
             if (sequenceId == 1)
             {
                 var lastInitSubmission = application.ApplicationData?.InitSubmissions.OrderByDescending(sub => sub.SubmittedAt).FirstOrDefault();
-                var contactToNotify = await _contactRepository.GetContact(lastInitSubmission?.SubmittedBy);
+                var contactToNotify = await _contactRepository.GetContact(lastInitSubmission.SubmittedBy);
```

Comment:
Is this a wise update? Code earlier indicates lastInitSubmission may be null

Reply:
Added a null check